server/.env

```diff
@@ -0,0 +1 @@
+DB_URL="mongodb+srv://wosen:wosen1122@cluster0.pszmqa3.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
For v2, please make sure .env and similar files are listed in your .gitignore, and remove the cached versions so you don't have this private information uploaded to GitHub.